Automattic/woocommerce-payments
# Fix failing unit test due to Woo default currency change.

#### Changes proposed in this Pull Request

Because [since Woo 5.3.0, the default currency is USD](https://github.com/woocommerce/woocommerce/commit/55ab8cb966c692685346db302625735cabb66f83#diff-b4fc4cdb5fd7e76a9e2bbb58f5e10621b636ace1f17cee96e0e0b2ca860bf5acL484-R484), some of our unit tests fail. This PR is to assume default currency is USD and set it so for Woo < 5.3.0.

#### Testing instructions

* Run unit test on base branch should fail.
* Run unit test on this branch should be successful.

-------------------
